starlite/testing/base/transport_base.py

```diff
@@ -77,17 +77,18 @@ async def receive() -> "ReceiveMessage":
     def create_send(request: "Request", context: SendReceiveContext) -> "Send":
         async def send(message: "Message") -> None:
             if message["type"] == "http.response.start":
-                assert not context["response_started"], 'Received multiple "http.response.start" messages.'
+                if context["response_started"]:
+                    raise AssertionError('Received multiple "http.response.start" messages.')
                 context["raw_kwargs"]["status_code"] = message["status"]
                 context["raw_kwargs"]["headers"] = [
                     (k.decode("utf-8"), v.decode("utf-8")) for k, v in message.get("headers", [])
                 ]
                 context["response_started"] = True
             elif message["type"] == "http.response.body":
-                assert context["response_started"], 'Received "http.response.body" without "http.response.start".'
-                assert not context[
-                    "response_complete"
-                ].is_set(), 'Received "http.response.body" after response completed.'
+                if not context["response_started"]:
+                    raise AssertionError('Received "http.response.body" without "http.response.start".')
```

Comment:
please revert this change ans use asserts. You can ignore the flake8 errors with a noqa comment. You are basically implementing the asserts manually here.